.eslintrc.cjs

```diff
@@ -41,6 +41,7 @@ module.exports = {
         },
       },
     ],
+    'jsdoc/no-undefined-types': 'off',
```

Comment:
this rule sometimes treats the type that actually exists as an `undefined` type. better to disable this rule so we can use specific type that declared as `undefined` by this rule